onnxruntime/contrib_ops/cpu/transformers/logits_processor.cc

```diff
@@ -105,7 +106,8 @@ void NoRepeatNGramLogitsProcessor<T>::Process(const ISequences* sequences,
     gsl::span<const int64_t> prefix = sequence.subspan(sequence.length() - prefix_length);
     ORT_ENFORCE(prefix.length() == prefix_length);
 
-    std::unordered_set<int64_t> blocked_word_ids;
+    InlinedHashSet<int64_t> blocked_word_ids;
+    blocked_word_ids.reserve(sequence.size());
```

Comment:
>sequence.size() [](http://example.com/codeflow?start=29&length=15)

could it be `sequence.length() - ngram_size_ + 1`, based on the insertion loop condition?